thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -640,10 +627,9 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
             ifile->getTime(&createTime, &modifiedTime, &accessedTime);
             modifiedTime.serialize(mb);
             mb.append(partCrc);
-
-            mb.append(numLeafNodes);
-            mb.append(numBlobNodes);
-            mb.append(numBranchNodes);
+            mb.append(inactiveStats.getStatisticValue(StNumLeafCacheAdds));
+            mb.append(inactiveStats.getStatisticValue(StNumBlobCacheAdds));
+            mb.append(inactiveStats.getStatisticValue(StNumNodeCacheAdds));
```

Comment:
future: Is there a separate JIRA to remove these manual serializations? ..since they will now be serialized as part of the activity stats. and the deserializing and manual setting of @numLeafNodes, @numBranchNodes, @numBlobNodes is superfluous.

Reply:
I didn't want to change it in this PR as they are used in few other places.  Yes, I can remove these in a future jira.

Reply:
Future: https://hpccsystems.atlassian.net/browse/HPCC-33279